fish_command_timer.fish

```diff
@@ -123,7 +123,7 @@ else
   set -e fish_command_timer_enabled
 end
 
-set fish_command_timer_start_time
+set fish_command_timer_start_time (fish_command_timer_get_ts)
```

Comment:
Oh....I think I know what you mean. If you run `source fish_command_timer.sh` manually in the command line, then it does throw an error. But it won't raise an error when you have the `source` command in your `config.fish`.

Instead of setting an incorrect value here, could you wrap this in a `if`? e.g.,
```
if not set -q fish_command_timer_start_time
  set fish_command_timer_start_time
end
```

Thanks!